### PR TITLE
Fix viz plot compare evokeds

### PR DESCRIPTION
**Pull request by DominiqueMakowski**
Here's the small fix for plot_compare_evokeds (title fix + alpha adjustment exposition). 

I've removed the bayesian CI because 1) it was mainly exploratory, only tried out "*because why not*" and 2) it returned the almost same CI band than originally, but with much longer computation times.

Hope it's better to do the PR from a branch.

### CODE REVIEW

File: mne/viz/evoked.py [hunk 1]
```diff
@@ -1345,14 +1345,17 @@ def _ci(arr, ci):
...
     # then it becomes just:
     # return stats.t.interval(ci, loc=mean, scale=sigma, df=arr.shape[0])
-    return np.asarray([stats.t.interval(ci, arr.shape[0],
-                       loc=mean_, scale=sigma_)
-                       for mean_, sigma_ in zip(mean, sigma)]).T
+    ci_band = np.asarray([stats.t.interval(
+            ci, arr.shape[0], loc=mean_, scale=sigma_)
+            for mean_, sigma_ in zip(mean, sigma)]).T
+
+    return (ci_band)
```
Comment (jona-sassenhagen) [inline mne/viz/evoked.py]: I'm not sure this is an improvement ..? Especially considering it'll get replaced with a 1-liner once we update the minimum scipy.

File: mne/viz/evoked.py [hunk 2]
```diff
@@ -1599,10 +1605,11 @@ def plot_compare_evokeds(evokeds, picks=list(), gfp=False, colors=None,
     if ymin is None and (gfp is True or ch_type == 'grad'):
         ymin = 0.  # 'grad' and GFP are plotted as all-positive
 
-    # deal with dict/list of lists and the CI
-    if not isinstance(ci, np.float):
-        msg = '"ci" must be float, got {0} instead.'
-        raise TypeError(msg.format(type(ci)))
+    if ci is not None:
```
Comment (jona-sassenhagen) [inline mne/viz/evoked.py]: This is addressed in #4289, which was just merged.

**Comment by jona-sassenhagen** [comment]
Hey, good catch on the title, that needs to be fixed.

In what contexts have you found the default ci alpha to be lacking?

I'm sorry if this is confusing, but #4289, which was merged today, introduces a few updates to `plot_compare_evokeds`. Please check it out.

**Comment by DominiqueMakowski** [comment]
> I'm not sure this is an improvement ..? Especially considering it'll get replaced with a 1-liner once we update the minimum scipy.

@jona-sassenhagen it is not, looks like I missed the #4289 update.... How can I delete these changes an go back to the original? github won't let me "update from master" to feed in the new changes 😕 Maybe I could close this PR and you guys just add the title fix thing, if that works, in the next update?

As for the ci_alpha, I'm currently doing some ready-for-publication plots and I'd like to decrease the default alpha. But I don't know, is mne's philosophy to prevent major cosmetic changes and keep a certain visual identity? If so, this change can be removed also :sweat_smile:

**Comment by agramfort** [comment]
you can try to rebase.

to change fig properties you have the fig object returned so you should be
able
to change the look and feel without changing the mne-python code.

**Comment by jona-sassenhagen** [comment]
Still, if you think the current defaults for the ci alpha are not optimal, we need to do something about that.

**Comment by agramfort** [comment]
I let you converge to the "perfect" look and feel :)

**Comment by jona-sassenhagen** [comment]
>How can I delete these changes an go back to the original? github won't let me "update from master" to feed in the new changes 😕 Maybe I could close this PR and you guys just add the title fix thing, if that works, in the next update?

As Alex says, the proper way is probably to rebase, but I'm lazy so I'd just close this and open a new PR ...
But I think you should definitely address this issue in a PR, after all, you were the one who discovered it :D

> I don't know, is mne's philosophy to prevent major cosmetic changes and keep a certain visual identity? 

My point is, we should provide sensible default. Do you think the default alpha is too high per se? Or is it specific to your case?

**Comment by jona-sassenhagen** [comment]
@DominiqueMakowski  please reopen if you don't think this has been addressed, or is not adressed by #4526

**Pull request closed without merging**